cypress/integration/pages/onDemandTV/index.js

```diff
@@ -44,11 +44,7 @@ Object.keys(config)
             variant,
             isAmp: true,
           });
-          testsForAMPOnly({
-            service,
-            pageType,
-            variant,
-          });
+          testsForAMPOnly();
```

Comment:
This appears to be an empty function - https://github.com/bbc/simorgh/blob/latest/cypress/integration/pages/onDemandTV/testsForAMPOnly.js

Reply:
I've raised an issue to resolve this later https://github.com/bbc/simorgh/issues/7816 - don't want to include it as part of this PR